packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx

```diff
@@ -24,6 +54,72 @@ export const statuses = {
 
 export function IntegrationList() {
   const intl = useIntl()
+  const [showModal, setToggleModal] = React.useState<boolean>(false)
+  const [createClientInfo, setCreateClientInfo] = React.useState<boolean>(true)
+  const [generateClientInfo, setGenerateClientInfo] =
+    React.useState<boolean>(false)
+  const [clientName, setClientName] = React.useState<string>(EMPTY_STRING)
+  const [clientType, setClientType] = React.useState<string>(EMPTY_STRING)
+  const [selectedTab, setSelectedTab] = React.useState<string>(
+    WebhookOption.birth
+  )
+  const [selectedItems, setSelectedItems] = useState(['registration-details'])
+  const [selectedItemsNoPII, setSelectedItemsNoPII] = useState([
+    'registration-details-noPII'
+  ])
+  const [NoPII, setSelected] = React.useState(false)
+
+  const toggleModal = () => {
+    setToggleModal((prev: any) => !prev)
+  }
+
+  const changeModalInfo = () => {
+    setCreateClientInfo(!createClientInfo)
+    setGenerateClientInfo(!generateClientInfo)
+  }
+
+  const AlertSyled = styled(Alert)`
+    --color: rgb(56, 162, 157);
+    margin-top: 30px;
+    min-height: 5rem;
```

Comment:
If the color isn't in Alert type-props, could you confirm with @jpye-finch that should we add the color to Alert, or should the design use a different color? We should not hard code the color here

Reply:
okay..the actual colour this alert type isn't available in type-props..thats why i specified the colour code..  would communicate this with @jpye-finch  and update this.